portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,52 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+console.log("hello world");
+
+async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('fetchTutorial').innerText = quote;
```

Comment:
Is this function used to get the comments? If so - I'd recommend sticking to one name rather than introducing `quote`.

Reply:
Also `fetchTutorial` might be best named `commentsSection`

Reply:
this is old and i just hadn't deleted yet. I just removed it all